app/src/main/java/com/keylesspalace/tusky/fragment/AccountMediaFragment.kt

```diff
@@ -301,8 +301,15 @@ class AccountMediaFragment : BaseFragment(), Injectable {
             itemBgBaseHSV[2] = random.nextFloat() * (1f - 0.3f) + 0.3f
             holder.imageView.setBackgroundColor(Color.HSVToColor(itemBgBaseHSV))
             val item = items[position]
+
+            val maxW = context!!.resources.getInteger(R.integer.media_max_width)
+            val maxH = context!!.resources.getInteger(R.integer.media_max_height)
```

Comment:
Please never `context!!` this can cause crashes. You can get a non-null context from a view, so here `holder.imageView.context`